src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/BeneficialOwnerGovernmentOrPublicAuthorityDto.java

```diff
@@ -50,6 +55,18 @@ public class BeneficialOwnerGovernmentOrPublicAuthorityDto {
     @JsonProperty(NON_LEGAL_FIRM_MEMBERS_NATURE_OF_CONTROL_TYPES_FIELD)
     private List<NatureOfControlType> nonLegalFirmMembersNatureOfControlTypes;
 
+    @JsonProperty(NON_LEGAL_FIRM_CONTROL_NATURE_OF_CONTROL_TYPES_FIELD)
+    private List<NatureOfControlType> nonLegalFirmControlNatureOfControlTypes;
+
+    @JsonProperty(TRUST_NATURE_OF_CONTROL_TYPES_FIELD)
+    private List<NatureOfControlType> trustNatureOfControlTypes;
```

Comment:
trustNatureOfControlTypes should be trustControlNatureOfControlTypes

Reply:
See question above.

Reply:
updated